integration/js/VideoTest.js

```diff
@@ -26,15 +29,27 @@ class VideoTest extends SdkBaseTest {
       // todo: add step to check filter content
       await ClickVideoFilterButton.executeStep(this, session);
     }
-    await ClickVideoButton.executeStep(this, session);
-    await LocalVideoCheck.executeStep(this, session, 'VIDEO_ON');
-    await WaitForRemoteParticipantsToTurnVideoOn.executeStep(this, session);
-    await RemoteVideoCheck.executeStep(this, session, 'VIDEO_ON');
-    await WaitForRemoteVideoCheckToComplete.executeStep(this, session);
-    await ClickVideoButton.executeStep(this, session);
-    await LocalVideoCheck.executeStep(this, session, 'VIDEO_OFF');
-    await WaitForRemoteParticipantsToTurnVideoOff.executeStep(this, session);
-    await RemoteVideoCheck.executeStep(this, session, 'VIDEO_OFF');
+    for (let i = 0; i < videoToggleCount; i++) {
+        // Note since these wait functions will be called twice, we reset the respective variable after each one
```

Comment:
Not super pretty imo but is simply a requirement of the way this state is held, couldn't find a cleaner way (reset in step seems like a weird side effect, creating a reset function for every variable seemed like overkill) that didn't require more significant refactoring of these tests.